src/Handlebars.Net.Helpers/HandlebarsHelpers.cs

```diff
@@ -100,7 +103,7 @@ private static void RegisterHelper(HandlebarsHelpersOptions helperOptions, IHand
             {
                 handlebarsContext.RegisterHelper(helperName, (writer, context, arguments) =>
                 {
-                    object value = InvokeMethod(helperOptions, helperName, methodInfo, arguments, obj);
+                    object? value = InvokeMethod(helperOptions, helperName, methodInfo, arguments, obj);
```

Comment:
Idea for future improvements: you can use class per helper and implement `IHelperDescriptor<THelperOptions>` interface. This should make code simpler.

Reply:
For future